src/events/AudioEvents.hxx

```diff
@@ -17,6 +17,88 @@ struct Coordinate3D
   double x, y, z;
 };
 
+/**
+ * @brief a standard reverb effect
+ */
+struct StandardReverbProperties
+{
+	//AL_REVERB_DENSITY, description: coloration of the late reverb, 
+	//range: 0.0 to 1.0.
+	double flDensity = 1.0;
+	
+	//AL_REVERB_DIFFUSION,  description: echo density in the reverberation decay, 
+	//range: 0.0 to 1.0
+	double flDiffusion = 1.0;
+	
+	//AL_REVERB_GAIN,   description: master volume control for the reflected sound, 
+	//range: 0.0 to 1.0
+	double flGain = 0.32;
+	
+	//AL_REVERB_GAINHF,   description: attenuation it at high frequencies,
+	//range: 0.0 to 1.0
+	double flGainHF = 0.89;
+	
+	//AL_REVERB_DECAY_TIME,  description: reverberation decay time, 
+	//range: 0.1 to 20.0
+	double flDecayTime = 1.49;
+	
+	//AL_REVERB_DECAY_HFRATIO,  description: ratio of high-frequency decay time relative to the time set by Decay Time,
+	// range: 0.1 to 2.0
+	double flDecayHFRatio = 0.83;
+	
+	//AL_REVERB_REFLECTIONS_GAIN,  description: controls the overall amount of initial reflections relative to the Gain property
+	//, range: 0.0 to 3.16
+	double flReflectionsGain = 0.05;
+	
+	//AL_REVERB_REFLECTIONS_DELAY,  description: amount of delay between the arrival time of the direct path from the source to the first reflection from the source
+	//range: 0.0 to 0.3
+	double flReflectionsDelay = 0.007;
+	
+	//AL_REVERB_LATE_REVERB_GAIN,  description: overall amount of later reverberation relative to the Gain property
+	//range: 0.0 to 10.0
+	double flLateReverbGain = 1.26;
+	
+	//AL_REVERB_LATE_REVERB_DELAY,  description: begin time of the late reverberation relative to the time of the initial reflection
+	//range: 0.0 to 0.1
+	double flLateReverbDelay = 0.011;
+	
+	//AL_REVERB_AIR_ABSORPTION_GAINHF,  description: the distance-dependent attenuation at high frequencies caused by the propagation medium
+	//range: 0.892 to 1.0
+	double flAirAbsorptionGainHF = 0.994;
+	
+	//AL_REVERB_ROOM_ROLLOFF_FACTOR,  description: attenuate the reflected sound,
+	//range: 0.0 to 10.0
+	double flRoomRolloffFactor = 0.0;
+	
+	//AL_REVERB_DECAY_HFLIMIT,  description: ,range:
+	//int iDecayHFLimit;
```

Comment:
use 3 slashes for doxygen documentation.

@Ercadio I don't know the syntax to properly document a struct in doxygen. can you help out here?